app/models/manageiq/consumption/showback_price_plan.rb

```diff
@@ -34,6 +34,14 @@ def calculate_cost(event)
     tc
   end
 
+  def bill(start_t, end_t)
+    t = Money.new(0)
+    ManageIQ::Consumption::ShowbackPool.where(:state => "CLOSED",:resource => resource).each do |pool|
```

Comment:
start_t?
end_t?

You need to consider also this month: .* pools processing + 1 pool open per resource